docs/layouts/shortcodes/source-integration/validate.md

```diff
@@ -11,7 +11,20 @@ platform integration:validate
 ###{{ if eq $source "Bitbucket" }}#{{ end }} Add the webhook manually
 
 If the integration was added with the correct permissions, the necessary webhook is added automatically.
-If you see a message that the webhook wasn't added, add one manually:
+If you see a message that the webhook wasn't added, add one manually.
+
+{{ if eq $source "Bitbucket" }}
+To configure a webhook on a {{ $source }} repository,
+you need to have Admin [user permissions](https://support.atlassian.com/bitbucket-cloud/docs/manage-webhooks/).
+{{ end }}
+{{ if eq $source "GitHub" }}
+To configure a webhook on a {{ $source }} repository,
+you need to have Admin [user permissions](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization#permissions-for-each-role).
+{{ end }}
+{{ if eq $source "GitLab" }}
+To configure a webhook on a {{ $source }} repository,
+you need to have Maintainer or Owner [user permissions](https://docs.gitlab.com/ee/user/permissions.html).
+{{ end }}
```

Comment:
@chadwcarlson I'm guessing there's a way to only change the type of user and link to the appropriate docs based on the $source, but my all my attempts failed. Please let me know if this can be optimized and how.

Reply:
@AnouckColson this would be what I would try to start with.